PyDSS/element_fields.py

```diff
@@ -0,0 +1,119 @@
+ELEMENT_FIELDS = {
```

Comment:
Should Circuits also be added?  The test project I added exports these properties for Circuits:  "TotalPower", "LineLosses", "Losses", "SubstationLosses"

Reply:
done